versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 0.14.0"
```

Comment:
@milldr without that change outputs can't be marked sensitive, and then we definitely need the SSM component. But see [Here](https://github.com/cloudposse/terraform-aws-datadog-lambda-forwarder/pull/48#discussion_r1128611927), it looks like CloudPosse is ok with moving forward from 0.13

Reply:
Check with Erik please, I think if we're good to move past 0.13 then this PR should jump to 1.3.0 or even 1.4.2 so that we don't face this version upgrading problem in the future in this module.

Reply:
okay I've checked and it's no problem to bump that version. But in order for the tests to pass we need to update the version provided with testing

https://github.com/cloudposse/terraform-aws-documentdb-cluster/blob/master/examples/complete/versions.tf#L2